### PR TITLE
integration: don't create a factory for each test case

### PR DESCRIPTION
**Pull request by avagin**

### CODE REVIEW

File: vendor/src/github.com/docker/libcontainer [hunk 1]
```diff
@@ -0,0 +1 @@
+../../../../
```
Comment (dqminh) [inline vendor/src/github.com/docker/libcontainer]: hmm i think we dont need this. If the desired state is to have a compiled nsinit binary, there's `make binary` which installed nsinit binary into `./bundle`

Comment (avagin) [inline vendor/src/github.com/docker/libcontainer]: @dqminh I don't want to execute a docker container to compile nsinit or execute tests. Why is it bad to have this link?

Comment (avagin) [inline vendor/src/github.com/docker/libcontainer]: @dqminh I agree that this commit should not be in this series. I've removed it.

**Comment by LK4D4** [comment]
LGTM

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**